test/test_overrides.py

```diff
@@ -1001,5 +1001,21 @@ def test_rnn(self):
         model(input)
 
 
+class TestDisabledTorchFunction(TestCase):
+    # Regression test for gh-64687
+    def test_parameter_does_not_prevent_dispatch(self):
+        class MyTensor():
+            def __torch_function__(self, func, types, args=(), kwargs=None):
+                return "called"
+
+        t1 = MyTensor()
+        t2 = torch.nn.Parameter(torch.rand(2, 2))
```

Comment:
nit: given our discussion, Parameter might not use the disabled torch function in the future. Should we consider using a local class that uses that feature for this test?

Reply:
nah, if we nuke disabled torch function from Parameter, I'm going to guess that this will be globally nuked

Reply:
So the default implementation should always be there and the user should explicitly provide a no-op if they want one?

Reply:
yeah. Well, that's the idea anyway, I'm not sure if it will actually work